### PR TITLE
Sample files for 2016-Dev0

### PR DESCRIPTION
**Pull request by dgault**
This PR contains updated versions of the 2015 sample files, converted to match the new 2016-Dev0 schema. Alongside the 2015 samples there are additionally 2 sample folder files converted to OME XML to add coverage for the folder model updates.

### CODE REVIEW

File: components/specification/samples/2016-01/minimum-specification.ome.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OME xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-DEV0
+                             http://www.openmicroscopy.org/Schemas/OME/2016-DEV0/ome.xsd">
+  <Image ID="Image:1" Name="Name92">
+    <AcquisitionDate>2009-03-16T15:41:51.0Z</AcquisitionDate>
```
Comment (sbesson) [inline components/specification/samples/2016-01/minimum-specification.ome.xml]: While we are updating samples, `AcquisitionDate` is now optional and should likely be dropped from the minimum specification example.

Comment (hflynn) [inline components/specification/samples/2016-01/minimum-specification.ome.xml]: That reminds me, we also need to update https://www.openmicroscopy.org/site/support/ome-model/specifications/minimum.html. Shall I open another docs PR for that?

Comment (sbesson) [inline components/specification/samples/2016-01/minimum-specification.ome.xml]: Once we have the final version of the samples finalized and tested, we will certainly need to review the hard-coded code bits in the model/formats documentation.

Comment (hflynn) [inline components/specification/samples/2016-01/minimum-specification.ome.xml]: Ok, I will add that to the checklist on https://trello.com/c/gskA0JKM/65-schema-samples-generation

**Comment by sbesson** [comment]
Trying to generate configuration files for this sample set, I realized all remaining `MarkedStart`/`MarkerEnd` using `Circle` or `Square` need to be updated to use `Arrows` (or nothing).

**Comment by sbesson** [comment]
With https://github.com/openmicroscopy/bioformats/pull/2459, these OME-XML samples files can now be updated to use `2016-06` instead of `2016-DEV0`.

**Comment by sbesson** [comment]
A couple of issues reported by the various C++ builds, e.g. https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge/BUILD_TYPE=Release,node=marlin/223:
- some dropped attributes/enums remain on certain shapes incl. `Visible` and `Linecap` attributes and a couple of `Circle` enumerations
- some ROI IDs have lost their `ROI:` prefix and fail the XML validation 
- in `transformations-downgrade.ome.xml` and `transformations-upgrade.ome.xml` the `Transform` elements should be subelements of the corresponding `Shape` element instead of the `ROI` element
- in the `folders-*` samples, the `TiffData` blocks should be removed in favor of the existing `BinData` block. Also this sample could be reindented and `Creator="OME Bio-Formats 5.2.0-SNAPSHOT"` removed.

**Comment by sbesson** [comment]
Still a few errors left - see https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge/BUILD_TYPE=Release,node=marlin/224/console. I can work on some fix commits later today
or in the week-end if no-one gets to it.

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge/229/, https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge-superbuild/260/ and https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge-win-superbuild/285/ green with the latest state of this PR. :+1:

**Pull request merged**